File: Programming/Lv.1/문자열 내 p와 y의 개수/solution.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12916?language=python3
def solution(s):
    c = 0
    
    for l in s.lower():
        if l == 'p': c += 1
        elif l == 'y': c -= 1

    return True if c == 0 else False

# count 함수가 있구나
# def numPY(s):
#     # 함수를 완성하세요
#     return s.lower().count('p') == s.lower().count('y')

File: Programming/Lv.1/문자열 내 p와 y의 개수/solution.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12916
function solution(s) {
  let count = 0;
  s.toLowerCase()
    .split("")
    .forEach((v) => {
      if (v === "p") count += 1;
      else if (v === "y") count -= 1;
    });

  return count === 0 ? true : false;
}
